src/components/use/dimension-list.jsx

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+import usePromise from 'react-use-promise';
+
+export default function useModel(app, def) {
```

Comment:
Why can't this just use `useModel` with a dimension list definition? Shouldn't have to duplicate the `useModel` code.